### ISSUE TITLE
Set ShowOnCreate = false for all Id properties with auto-generation turned on

### ISSUE THREAD

**Issue by korzh** [KorzhCom/EasyData#96]
If we define a model class like this:

```
public class Language
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string ShortName { get; set; }
}
```
this `Id` property here becomes a primary key with auto-generation. The fields of this kind can't be edited, so we have to detect this situation and mark corresponding meta attributes as "Non-editable".

**Comment by antifree** [comment]
It's strange, by default in the UI we do not allow editing primary keys, so there is no need to add such a feature

**Comment by antifree** [comment]
To hide on Create

**Comment by antifree** [comment]
Resolved in commit f90ea57739aff330ffdc5e105e2235f87d8fd5fc

**Comment by korzh** [comment]
Done in 1.4.1-rc02
